test/libraries/Safe150Migration.spec.ts

```diff
@@ -0,0 +1,771 @@
+import { expect } from "chai";
+import hre, { ethers, deployments } from "hardhat";
+import { AddressZero } from "@ethersproject/constants";
+import { getSafeWithSingleton, migrationContractTo150, getSafeSingletonAt, getMock } from "../utils/setup";
+import deploymentData from "../json/safeDeployment.json";
+import safeRuntimeBytecode from "../json/safeRuntimeBytecode.json";
+import { executeContractCallWithSigners } from "../../src/utils/execution";
+
+const SAFE_SINGLETON_150_ADDRESS = "0x88627c8904eCd9DF96A572Ef32A7ff13b199Ed8D";
```

Comment:
Shouldn't we be able to calculate this based on the runtime bytecode?

Reply:
We can, but I'm not sure why, since after 1.5.0 is released and the addresses known this doesn't have to be adjusted any more